src/main/java/crazypants/enderio/machine/farm/TileFarmStation.java

```diff
@@ -424,23 +450,50 @@ protected void doTick() {
 
     if(!isOpen(bc)) {
       IHarvestResult harvest = FarmersCommune.instance.harvestBlock(this, bc, block, meta);
-      if(harvest != null) {
-        if(harvest.getDrops() != null) {
-          PacketFarmAction pkt = new PacketFarmAction(harvest.getHarvestedBlocks());
-          PacketHandler.INSTANCE.sendToAllAround(pkt, new TargetPoint(worldObj.provider.dimensionId, bc.x, bc.y, bc.z, 64));
-          for (EntityItem ei : harvest.getDrops()) {
-            if(ei != null) {
-              insertHarvestDrop(ei, bc);
-              if(!ei.isDead) {
-                worldObj.spawnEntityInWorld(ei);
-              }
+      if(harvest != null && harvest.getDrops() != null) {
```

Comment:
I get what you did here, but it makes it very difficult to see what changed. I'd prefer if you didn't change indentation unless necessary.